core/resources/shaders/point.vs

```diff
@@ -26,11 +18,19 @@ uniform float u_meters_per_pixel;
 
 #pragma tangram: uniforms
 
-varying float v_alpha;
-varying vec2 v_texcoords;
+attribute vec2 a_uv;
+attribute LOWP vec2 a_position;
+attribute LOWP vec2 a_screenPosition;
+attribute LOWP float a_alpha;
+attribute LOWP float a_rotation;
+attribute LOWP vec4 a_color;
+attribute LOWP vec4 a_stroke;
+
 varying vec4 v_color;
```

Comment:
Just wondering about all the LOWP, 2^-8 precision is not sufficient for vertices, but most drivers will give you more precision anyway. Have you actually tried it on a device that uses the minimal requested precision?  Galaxy S6 seems to use the hint (maybe only for one particular driver version though)

https://www.khronos.org/registry/gles/specs/2.0/GLSL_ES_Specification_1.0.17.pdf (Precision qualifiers)